Example/Controllers/SwiftUI/Features/Settings/Features/HostAppThemeColorsView.swift

```diff
@@ -0,0 +1,88 @@
+import SwiftUI
+import MiniApp
+
+struct HostAppThemeColorsView: View {
+    @ObservedObject var viewModel: MiniAppSettingsViewModel
+    @State var primaryColor: String = ""
+    @State var secondaryColor: String = ""
+    @State private var pColor = Color.red
+    @State private var sColor = Color.blue
+    @State private var alertMessage: MiniAppAlertMessage?
+
+    var body: some View {
+        Form {
+            Section(header: Text("Enter color as Hex values")) {
```

Comment:
We can either have this or the footer text. I see we are mentioning them to enter Hex values in two places

Reply:
The section header is removed as we are keeping the footer note.